app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(comm: str) -> None:
+    parts = comm.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    src, dst = parts[1], parts[2]
```

Comment:
It is possible to use unpacking as well to avoid indexing
```suggestion
    _, src, dst = parts
```